File: problems/2923-find-champion-i/solution.c
int findChampion(int** grid, int gridSize, int* gridColSize) {
    for (int i = 0; i < gridSize; i++) {
        int champion = 1;
        for (int j = 0; j < gridSize; j++) {
            if (i != j && grid[j][i] == 1) {
                champion = 0;
                break;
            }
        }
        if (champion) {
            return i;
        }
    }
    return -1;
}

File: problems/2923-find-champion-i/solution.py
class Solution:
    def findChampion(self, grid: List[List[int]]) -> int:
        n = len(grid)
        for i in range(n):
            is_champion = True
            for j in range(n):
                if i != j and grid[j][i] == 1:
                    is_champion = False
                    break
            if is_champion:
                return i